src/test/java/uk/gov/digital/ho/pttg/SmokeTest.java

```diff
@@ -1,11 +1,63 @@
 package uk.gov.digital.ho.pttg;
 
+import com.google.gson.*;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.junit.Before;
 import org.junit.Test;
+import uk.gov.digital.ho.pttg.api.FinancialStatusRequest;
+import uk.gov.digital.ho.pttg.api.HttpResponse;
+
+import java.lang.reflect.Type;
+import java.time.LocalDate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.fail;
 
 public class SmokeTest {
 
+    private final String IP_API_PATH = "/incomeproving/v3/individual/financialstatus";
+
+    private String ipApiRootUrl;
+    private CloseableHttpClient httpClient;
+    private SimpleHttpClient simpleHttpClient;
+    private Gson gson;
+
+    @Before
+    public void setUp() {
+        ipApiRootUrl = getMandatoryEnvVar("IP_API_ROOT_URL");
+        simpleHttpClient = new SimpleHttpClient();
+        gson = new GsonBuilder().registerTypeAdapter(LocalDate.class, new LocalDateSerializer())
+                                .create();
+    }
+
     @Test
-    public void testStartup() {
-        // TODO EE-6825 Tests yet to be added - for now it is sufficient to check that the tests run
+    public void testIpApi() {
+        String body = buildRequest();
+        System.out.println(body);
+        HttpResponse response = simpleHttpClient.post(ipApiRootUrl + IP_API_PATH, body);
+        System.out.println(response.body());
+
+        assertThat(response.statusCode()).isEqualTo(404);
```

Comment:
Is this a genuine not found, or a not matched by either HMRC or DWP?

Reply:
Sorry, I just realised you're testing IPS, not RPS.  Please ignore!

Reply:
A genuine not found. Obviously 404 can mean a completely wrong URL but the next line checks that the response looks like what pttg-ip-api returns for a unmatched identity.

Reply:
I can add more to the body check to be certain if you think that's appropriate?

Reply:
No it's fine - I was thinking about RPS where a 404 means not found rather than not matched.